build.sbt

```diff
@@ -10,7 +10,6 @@ scalacOptions ++= Seq(
   "-feature",
   "-unchecked",
   "-language:higherKinds",
-  "-Yno-adapted-args",
```

Comment:
Gone in 2.13, but apparently will be back in 2.13.2: https://github.com/scala/bug/issues/11110#issuecomment-526592068